CryptoTechReminderSystem.Test/TestDouble/GetLateDevelopersStub.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using CryptoTechReminderSystem.Boundary;
+using CryptoTechReminderSystem.UseCase;
+
+namespace CryptoTechReminderSystem.Test.TestDouble
+{
+    public class GetLateDevelopersStub : IGetLateDevelopers
+    {
+        private readonly List<string> _lateDevelopers;
+        
+        public GetLateDevelopersStub(List<string> lateDevelopers = null)
+        {
+            if (lateDevelopers == null)
```

Comment:
```suggestion
            if (lateDevelopers == null) lateDevelopers = DefaultList();
```

Reply:
now you can remove the else and make this method way shorter

Reply:
We've resolved this in https://github.com/chubberlisk/crypto_tech/pull/17/commits/e938a37d80ef091e14eab57a6c80e21c868d846a.